src/AbstractWebTestCase.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Smart\StandardBundle;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Liip\TestFixturesBundle\Services\DatabaseToolCollection;
+use Liip\TestFixturesBundle\Services\DatabaseTools\AbstractDatabaseTool;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+
+abstract class AbstractWebTestCase extends WebTestCase
+{
+    protected ?KernelBrowser $client = null;
+    protected ?EntityManagerInterface $entityManager = null;
+    protected AbstractDatabaseTool $databaseTool;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->client = self::createClient();
+        $this->entityManager = static::getContainer()->get(EntityManagerInterface::class);
+        // https://github.com/liip/LiipTestFixturesBundle/blob/2.x/UPGRADE-2.0.md
+        // @phpstan-ignore-next-line
+        $this->databaseTool = static::getContainer()->get(DatabaseToolCollection::class)->get();
+    }
+
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        // https://github.com/liip/LiipTestFixturesBundle/pull/196/files
+        unset($this->databaseTool);
+
+        // avoid memory leaks
+        if ($this->entityManager != null) {
+            $this->entityManager->close();
+            $this->entityManager = null;
+        }
+    }
+
+    protected function getParameter(string $name): mixed
+    {
+        return static::getContainer()->getParameter($name);
+    }
+
+    protected function getFixturesDir(): string
+    {
+        return $this->getTestPath() . '/fixtures';
+    }
+
+    protected function getMinimalFixturesDir(): string
+    {
+        return $this->getTestPath() . '/../fixtures/minimal';
+    }
+
+    /**
+     * Return object property who can be private
+     * https://www.yellowduck.be/posts/test-private-and-protected-properties-using-phpunit
+     */
+    public static function getProperty(object $object, string $property): mixed
+    {
+        $reflectedClass = new \ReflectionClass($object);
+        $reflection = $reflectedClass->getProperty($property);
+        $reflection->setAccessible(true);
+
+        return $reflection->getValue($object);
+    }
+
+    /**
+     * Return object method who can be private
+     * https://stackoverflow.com/questions/249664/best-practices-to-test-protected-methods-with-phpunit
+     * @param class-string $class
+     */
+    protected static function getMethod(string $name, string $class): \ReflectionMethod
+    {
+        $method = (new \ReflectionClass($class))->getMethod($name);
+        $method->setAccessible(true);
+
+        return $method;
+    }
+
+    protected function getTestPath(): string
```

Comment:
@lfortunier rename the method as getProjectTestDir and move before getFixturesDir